bluephos/parameters/threshold.json

```diff
@@ -0,0 +1,9 @@
+{
+    "_comment": "Threshold parameters for the BluePhos Discovery Pipeline",
+    "thresholds": {"t_nn": 1.5, "t_ste": 1.5, "t_ed": 0.3},
+    "_descriptions": {
+        "t_nn": "Threshold for the neural network (NN) score. Determines the maximum allowed absolute value for the NN score to consider a candidate.",
```

Comment:
might be a good place to document this rather in the help string of argparse, or in the module docstring - or even actually in module docstring and set argparse description from that - was even thinking that might be a good default in the cli utilities.

Reply:
Thank you for the suggestion. After consideration, I moved the threshold definition into the CLI for better portability and flexibility.

Reply:
Thanks for the comments!